### PR TITLE
Update pt_BR.po

### PR DESCRIPTION
**Pull request by ojmarcelino**
Atualizada linguagem para um padrão mais natural ao usuário.

### CODE REVIEW

**Comment by keysym** [comment]
Hi @ojmarcelino!

I personally think we should base our translation on the already existing pacman translation, so we get consistent between both. As example, [pacman translates "Do you want to..." -> "Deseja..."](https://gitlab.archlinux.org/pacman/pacman/-/blob/master/src/pacman/po/pt_BR.po#L170-175), not as ["Você quer..." as proposed](https://github.com/Morganamilo/paru/pull/450/files#diff-c665e9114d0107a20170b2c7a27f12bfa99c3d1fa8e550888ecc6d51631ebc97R40-R42).

But [translating "Repository" to the full word "Repositório"](https://github.com/Morganamilo/paru/pull/450/files#diff-c665e9114d0107a20170b2c7a27f12bfa99c3d1fa8e550888ecc6d51631ebc97L490-R492) is a change that I fully agree with! Great catch!

Do you think we should revise for it to be more standardized with pacman?
I'm also pinging @DatCrazyDelphox cause he was the first pt_BR committer!

**Comment by DatCrazyDelphox** [comment]
Its fine by me, no prob

**Review by ojmarcelino — COMMENTED** [review]
Great, @keysym, fine by me too, no problem to leave those translations to the "nearest" as possible. Thanks.

btw, "repo" is a pretty more to a slang than some tech jargon imho.

**Comment by DatCrazyDelphox** [comment]
But since i'm the original committer, I think my name should go in the top part and @ojmarcelino should be on that last translated field

probably an oversight on my part

**Comment by ojmarcelino** [comment]
great, send them over, I just wanna help, not matter to me the "sequence"

> But since i'm the original committer, I think my name should go in the top part and @ojmarcelino should be on that last translated field
> 
> probably an oversight on my part

File: po/pt_BR.po [hunk 1]
```diff
@@ -381,15 +381,15 @@ msgstr "nenhum alvo especificado (use -h para ajuda)"
 #: src/install.rs:434 src/install.rs:461 src/search.rs:324 src/search.rs:360
 #: src/lib.rs:227
 msgid " there is nothing to do"
-msgstr " Nada para fazer"
+msgstr " não há nada a fazer"
```
Comment (DatCrazyDelphox) [inline po/pt_BR.po:384]: This one is part of pacman's translation, so, it doesn´t need to change

File: po/pt_BR.po [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # SOME DESCRIPTIVE TITLE.
 # Copyright (C) YEAR THE paru'S COPYRIGHT HOLDER
 # This file is distributed under the same license as the paru package.
-# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+# Juliano Marcelino (ojmarcelino), 2021.
```
Comment (DatCrazyDelphox) [inline po/pt_BR.po:4]: here you can place `Pedro Liberatti <ferrahwolfeh@protonmail.com>, 2021`

and place your name in the last translator field

**Review by DatCrazyDelphox — CHANGES_REQUESTED** [review]
just this part really

**Comment by DatCrazyDelphox** [comment]
ideally, the complete file with all changes should look like this

[pt_BR.txt](https://github.com/Morganamilo/paru/files/6734319/pt_BR.txt)

you can update this pull request accordingly

**Comment by ojmarcelino** [comment]
sure, I've just made few changes, like "secção" in pt-pt is equivalent to "seção" in pt-br, so this and another changes are covered in my pr, following the latest defined in our country. "Providência" as provides, same thing, in pt-br is "provê" (not sure about pt-pt in this case). Mantenedor is more related to a university campus sponsor, who keeps the university running, one of stakeholders, if applied.
latest file guess will be this one, [pt_BR.txt](https://github.com/Morganamilo/paru/files/6734680/pt_BR.txt)

**Review by ojmarcelino — COMMENTED** [review]
added translators in deed.

**Comment by keysym** [comment]
Since this also fixes "Secção" -> "Seção", I'll be closing #446.

I'm in working hours... But later in the day I would like to make some proposals to your branch!

**Comment by keysym** [comment]
https://github.com/ojmarcelino/paru/pull/1

**Pull request merged**